src/libraries/System.Data.Common/tests/System/Data/SqlTypes/SqlXmlTest.cs

```diff
@@ -119,6 +96,82 @@ public void SqlXml_fromXmlReader_CreateReaderTest()
             Assert.Equal(xmlStr, xrdr.ReadOuterXml());
         }
 
+        [Theory]
+        [InlineData("element_whitespace-text.xml")]
+        [InlineData("root_qname.xml")]
+        [InlineData("sample_ecommerce.xml")]
+        [InlineData("sql_batch_request.xml")]
+        [InlineData("sql_batch_response.xml")]
+        [InlineData("sql_datatypes-1.xml")]
+        [InlineData("sql_datatypes-2.xml")]
+        [InlineData("sql_datatypes-3.xml")]
+        [InlineData("xmlns-1.xml")]
+        [InlineData("xmlns-2.xml")]
+        [InlineData("xmlns-3.xml")]
+        [InlineData("xmlns-4.xml")]
+        [InlineData("comments_pis.xml")]
+        [InlineData("element_content_growth.xml")]
+        [InlineData("element_nested-1.xml")]
+        [InlineData("element_nested-2.xml")]
+        [InlineData("element_nested-3.xml")]
+        [InlineData("element_single.xml")]
+        [InlineData("element_stack_growth.xml")]
+        [InlineData("element_tagname_growth.xml")]
+        [InlineData("element_types.xml")]
+        [InlineData("element_whitespace-modes.xml")]
+        public void SqlXml_fromXmlReader_TextXml(string filename)
+        {
+            string filepath = Path.Combine("TestFiles/SqlXml/TextXml", filename);
+
+            using FileStream xmlStream = new FileStream(filepath, FileMode.Open);
+            SqlXml sqlXml = new SqlXml(xmlStream);
+
+            // Reading XML stored as SQL Binary XML will result in using
+            // the XmlTextReader implementation
+            using XmlReader sqlXmlReader = sqlXml.CreateReader();
+
+            // Read to the end to verify no exceptions are thrown
+            while(sqlXmlReader.Read());
+        }
+
+        [Theory]
+        [InlineData("element_whitespace-text.bmx")]
```

Comment:
Should all of these be in runtime-assets instead?

Reply:
Also, do they need to all be listed explicitly, or should we just enumerate the test files directory and run the test with every file found?  That could either be in a MemberData function or it could be in the test itself if we don't care about having a distinct test execution for each file.

Reply:
> Should all of these be in runtime-assets instead?

This would address https://github.com/dotnet/runtime/pull/81878#issuecomment-1424364484

Reply:
Thanks. Since the files were all tiny, I had thought `runtime-assets` would be overkill. But based on the other comments, I'm moving them out there. Thanks for the heads-up about how this would have affected other efforts!